src/cmd/root.go

```diff
@@ -177,6 +178,13 @@ func rootHelp(cmd *cobra.Command, args []string) {
 }
 
 func rootRun(cmd *cobra.Command, args []string) error {
+	if len(args) == 0 {
+		rootFlags.assumeYes = true
+		rootFlags.enterShortcut = true
+
+		return enter(enterCmd, args)
+	}
+
 	var builder strings.Builder
 	fmt.Fprintf(&builder, "missing command\n")
```

Comment:
Is there a reason to keep the `missing command` message anymore?

Reply:
The intent was for this to be configurable. As that is not the intent, it is redundant.